capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -70,6 +72,11 @@ class ClubPage extends Component {
         .then(memberJson => memberJson && this.setState({ members: [...this.state.members, memberJson] }))
         .catch(e => console.log(e));
     }
+
+    fetch(`/api/meetings?clubID=${club.id}`)
+      .then(response => response.json()).then(meetings => this.setState({ meetings }))
```

Comment:
nit: Move the second `.then` down to the next line for readability.

Reply:
Will do!